mmv1/third_party/terraform/website/docs/r/google_project_service.html.markdown

```diff
@@ -45,15 +47,16 @@ The following arguments are supported:
 * `project` - (Optional) The project ID. If not provided, the provider project
 is used.
 
+* `disable_on_destroy` - (Optional) If `true` or unset, disable the service when the
+Terraform resource is destroyed. If `false`, the service will be left enabled when
+the Terraform resource is destroyed. Defaults to `true`. Most configurations should
```

Comment:
If most configurations should set this to false, should the default be changed in the future?

Reply:
We could consider it. Filed https://github.com/hashicorp/terraform-provider-google/issues/18058